.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+on:
+  release:
+    types:
+      - created
+jobs:
+  tests:
+    uses: ./.github/workflows/tests.yml
```

Comment:
Pretty cool that we can re-use the existing tests workflow to run all the tests first.

Unfortunately, we can't easily re-use the build from that workflow. There are ways, but it's not worth it, and we just re-run the build in the release job.